packages/macro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@popmenu/use-ssr-computation.macro",
-  "version": "1.0.3",
+  "version": "1.1.0",
```

Comment:
Updated the version because the new added option `skip` will not work with the older versions.